promenade/templates/genesis/etc/kubernetes/armada-loader/pki/armada-loader-key.pem

```diff
@@ -0,0 +1 @@
+{{ config.get(kind='CertificateKey', name='admin')['data'] }}
```

Comment:
I might be missing it, but where do these keys get generated from prior to the Jinja template getting created?